var/spack/repos/builtin/packages/sgpp/package.py

```diff
@@ -150,10 +151,7 @@ def build_args(self, spec, prefix):
         self.args.append("SG_DATADRIVEN={0}".format("1" if "+datadriven" in spec else "0"))
         self.args.append("SG_COMBIGRID={0}".format("1" if "+combigrid" in spec else "0"))
         self.args.append("SG_SOLVER={0}".format("1" if "+solver" in spec else "0"))
-
-        # Misc flag did not exist in older versions
-        if not spec.satisfies("@1.0.0:3.2.0"):
-            self.args.append("SG_MISC={0}".format("1" if "+misc" in spec else "0"))
+        self.args.append("SG_MISC={0}".format("1" if "+misc" in spec else "0"))
```

Comment:
Was the previous constraint wrong? Or is it just safe to add `SG_MISC` even if it doesn't exist/isn't used?

Reply:
Ah, gotcha. I missed that the older version weren't there anymore. Thanks!